File: C++/1차원 배열/최소, 최대.cc
#include <iostream>
using namespace std;

int main() {
    int n, tmp, min = 1000000, max = -1000000;
    cin >> n;
    
    for(int i = 0; i < n; i++) {
        cin >> tmp;
        if(min > tmp)
            min = tmp;
        if(max < tmp)
            max = tmp;
    }
    cout << min << " " << max;
}

File: Java/1차원 배열/최소, 최대.java
import java.io.*;
import java.util.StringTokenizer;

public class Main{
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
		int N = Integer.parseInt(br.readLine());
		int max = Integer.MIN_VALUE;
		int min = Integer.MAX_VALUE;
		StringTokenizer st = new StringTokenizer(br.readLine(), " ");
		while(st.hasMoreTokens()) {
			int num = Integer.parseInt(st.nextToken());
			min = Math.min(min, num);
			max = Math.max(max, num);
		}
		bw.write(min + " " + max);
		bw.flush();
    }
}